include/deal.II/grid/tria.h

```diff
@@ -1553,6 +1553,11 @@ class Triangulation : public Subscriptor
   virtual void set_mesh_smoothing (const MeshSmoothing mesh_smoothing);
 
   /**
+   * Return the mesh smoothing requirements that are obeyed.
+   */
+  virtual const Triangulation<dim, spacedim>::MeshSmoothing &get_mesh_smoothing() const;
```

Comment:
you can remove the qualification ``Triangulation<dim, spacedim>::`` here, right? Might look a bit cleaner.